mypyc/codegen/emitfunc.py

```diff
@@ -179,7 +182,7 @@ def visit_assign(self, op: Assign) -> None:
 
     def visit_load_int(self, op: LoadInt) -> None:
         dest = self.reg(op)
-        if is_c_int_rprimitive(op.type):
+        if is_int32_rprimitive(op.type) or is_int64_rprimitive(op.type):
```

Comment:
I've located the key to fix this. I guess that although I have not create any `LoadInt` with `int64_rprimitive` explicitly. And therefore all loaded int64 value would double and fail eventually.